module/Photo/src/Photo/Service/Admin.php

```diff
@@ -178,7 +178,8 @@ public function upload($files, $album)
             }
         } else {
             throw new \Exception(
-                $translator->translate('The uploaded file is not a valid image')
+                $translator->translate("The uploaded file is not a valid image \nError: " .
+                    json_encode($imageValidator->getMessages()))
```

Comment:
The manual (http://framework.zend.com/manual/current/en/modules/zend.validator.html) suggests something like
```
foreach ($imageValidator->getMessages() as $messageId => $message) {
        $translator->translate("Validation failure '$messageId': $message\n");
    }
```